src/components/RepoCard/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { AiOutlineStar } from 'react-icons/ai';
+import { VscRepoForked } from 'react-icons/vsc';
+import { TbCircleDot } from 'react-icons/tb';
+
+import styles from './index.module.css';
+
+const RepoCard = ({ repo }) => {
+    const {
+        name,
+        description,
+        stargazers_count,
+        open_issues_count,
+        forks_count,
+
+        created_at,
+        owner,
+    } = repo;
+    console.log(repo);
+
+    return (
+        <div className={styles.card}>
+            <div className={styles.firstSection}>
+                <a
+                    href={`https://github.com/alicekres/${name}`}
+                    className={styles.repoName}
+                    target="_blank"
+                >
+                    {name}{' '}
+                </a>
+                <p className={styles.description}>{description}</p>
+                <p className={styles.date}>Created on: {created_at} </p>
```

Comment:
Maybe change the date to a more UI-friendly format

Reply:
Yes, that's good advice. It's now done.